Exiled.API/Features/Player.cs

```diff
@@ -663,7 +666,11 @@ public float Health
         /// <summary>
         /// Gets or sets the player's maximum health.
         /// </summary>
-        public int MaxHealth { get; set; }
+        public float MaxHealth
```

Comment:
This should not change to an int, otherwise this becomes a breaking change.

Reply:
Some people wanna see the world burn